src/windows/network.rs

```diff
@@ -28,6 +42,25 @@ impl Networks {
             interfaces: HashMap::new(),
         }
     }
+
+    fn refresh_interfaces_addresses(&mut self) {
```

Comment:
This code is exactly the same across all implementations as far as I can see. Couldn't you put it into `network.rs` instead so the code isn't duplicated please?